lib/resources/cpan.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Richter
+
+# Usage:
+# describe cpan('DBD::Pg') do
+#   it { should be_installed }
+# end
+#
+
+module Inspec::Resources
+  class CpanPackage < Inspec.resource(1)
+    name 'cpan'
+    desc 'Use the `cpan` InSpec audit resource to test Perl modules that are installed by system packages or the CPAN installer.'
+    example "
+      describe cpan('DBD::Pg') do
+        it { should be_installed }
+      end
+    "
+
+    def initialize(package_name, perl_lib_path = nil)
+      @package_name = package_name
+      @perl_lib_path = perl_lib_path
+      @perl_cmd = 'perl'
+
+      # this resource is not supported on Windows
+      skip_resource 'The `cpan` resource is not supported on your OS yet.' if inspec.os.windows?
```

Comment:
Can you add a `return` to the beginning of this so we don't run an unnecessary command existence check for perl on Windows systems?